corehq/apps/domain/views.py

```diff
@@ -2209,6 +2232,26 @@ class DomainForwardingRepeatRecords(GenericTabularReport):
         'corehq.apps.reports.filters.select.RepeatRecordStateFilter',
     ]
 
+    def _make_cancel_payload_button(self, record_id):
```

Comment:
nit: do these need to be separate methods? I think there is a way you can compile this with a single `_make_payload_button` method... but maybe this is cleaner.